tests/e2e/tests/e2e/GitSsh.spec.ts

```diff
@@ -98,7 +102,9 @@ suite('Git with ssh workflow', async () => {
 
         data.metadata!.name = wsNameCheckPropagatingKeys;
         await testWorkspaceUtils.createWsFromDevFile(data);
+        await driverHelper.reloadPage();
         await driverHelper.navigateToUrl(workspacePrefixUrl + wsNameCheckPropagatingKeys);
+        await dashboard.waitPage();
```

Comment:
It's interesting to know why `waitWorkspaceAndIde()` is not enough and we need to wait on page before?

Reply:
Now, in new dashboard 'Create & Open' action doesn't open IDE immidiately. There is some waiting on dashboard page, then opens IDE. So this step is a normal behavior and also increases reliable of test.

![create-and-open-ws](https://user-images.githubusercontent.com/5408906/108088282-f5639100-7080-11eb-8328-e95b089f1899.png)

Reply:
I see. Thanks for explanation!